docs/gambit/mutation-verifier.md

```diff
@@ -112,9 +110,16 @@ We support `"staging" : true` as an alternative to `"staging" : "master"`.
 Omitting this key will cause verification to run locally
   (unless the run script has it).
 - `"cloud"` : if you instead want to run on the cloud environment you can provide the `--cloud` flag. You can also add the name of a specific branch.
-- `"use_cli_certora_run"` : Use CLI `certoraRun` rather than `certoraRun.py`. Expects a boolean and defaults to `false`.
+- `"use_certora_run_py"` : Use `certoraRun.py` rather than `certoraRun`. Expects a boolean and defaults to `false`.
 
 
+### Troubleshooting
+
+At the moment, there are a few ways in which `certoraMutate` can fail. Here we have some suggestions on how to troubleshoot when that happens. We are actively working on mitigating them.
+
+- Make sure the runscript you use to run the prover does not have the `--send_only` flag or any commented out (using `#`) lines.
+- Since Gambit requires you to provide the solidity compiler flags to compile the mutants, sometimes it might be useful to first identify what those flags should be. See {ref}`gambit-config` for more information. A strategy you can use is to first run `gambit` without going through certoraMutate. You likely have either `gambit-linux` or `gambit-macos` binaries in your path already if you are running the tool. Make a `foo.json` file and copy the content of the `"gambit":` field in it. Then run `gambit-OS mutate --json foo.json` to see what the issue could be.
```

Comment:
Since Gambit requires you to provide the solidity compiler flags to compile the mutants, it might be useful to first identify what those flags should be. See {ref}`gambit-config` for more information. A strategy you can adopt is:
1. Run `gambit` without going through certoraMutate. (You likely have either `gambit-linux` or `gambit-macos` 
binaries in your path already if you are running the tool).
 2. Make a `foo.json` file and copy the content of the `"gambit":` field in it. 
 3. Run `gambit-OS mutate --json foo.json` to try and identify the issue.